packages/forms/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@talend/react-forms",
   "description": "React forms library based on json schema form.",
-  "main": "lib/Form.js",
+  "main": "lib/index.js",
```

Comment:
I like this change, but I found a few imports that might need to be updated(?). Could you please check if they still should import Form.js, I'm not sure? 

- `import Form from '../Form';` in "ui\packages\forms\src\fields\CollapsibleFieldset.test.js"
- `import Form, { renderActionIcon, renderActions } from './Form';` in "ui\packages\forms\src\Form.test.js"
- `import Form from '../src/Form';` in "ui\packages\forms\stories\index.js"

Reply:
Those are internals and for this `Form.js` implementation so no need to be changed.
I have exposed some of the Form.js to not break all the things but yes, I should expose all of them in index.js to not add a breaking change.

The goal here is to enforce the import of UIForm from index and not from lib/UIForm